shared/teams/team/rows/invite-row/request/index.tsx

```diff
@@ -120,7 +117,7 @@ const TeamRequestRowNew = (props: Props) => {
           <Kb.Box2 direction="vertical" fullWidth={true}>
             <Kb.ConnectedUsernames type="BodyBold" colorFollowing={true} usernames={username} />
             <Kb.Box2 direction="horizontal" alignSelf="flex-start">
-              {(isNew || reset) && (
+              {reset && (
```

Comment:
I'm pretty sure this does the opposite of the title of the PR.

Reply:
🤦 thanks, fixed